server/sonar-server/src/main/java/org/sonar/server/startup/RenameIssueWidgets.java

```diff
@@ -0,0 +1,151 @@
+                .setWidgetId(widget.getId())
+                .setPropertyKey(FILTER_PROPERTY)
+                .setTextValue(hiddenDebt.getId().toString()));
+            widgetPropertiesToCreate.add(
+              new WidgetPropertyDto()
+                .setWidgetId(widget.getId())
+                .setPropertyKey(DISTRIBUTION_AXIS_PROPERTY)
+                .setTextValue("resolutions"));
+            updateWidget(session, widgetIdsWithPropertiesToDelete, widget);
+            break;
+          case "my_reviews":
+            widgetPropertiesToCreate.add(
+              new WidgetPropertyDto()
+                .setWidgetId(widget.getId())
+                .setPropertyKey(FILTER_PROPERTY)
+                .setTextValue(myUnresolvedIssues.getId().toString()));
+            updateWidget(session, widgetIdsWithPropertiesToDelete, widget);
+            break;
+          case "reviews_per_developer":
+            widgetPropertiesToCreate.add(
+              new WidgetPropertyDto()
+                .setWidgetId(widget.getId())
+                .setPropertyKey(FILTER_PROPERTY)
+                .setTextValue(unresolvedIssues.getId().toString()));
+            widgetPropertiesToCreate.add(
+              new WidgetPropertyDto()
+                .setWidgetId(widget.getId())
+                .setPropertyKey(DISTRIBUTION_AXIS_PROPERTY)
+                .setTextValue("assignees"));
+            updateWidget(session, widgetIdsWithPropertiesToDelete, widget);
+            break;
+          case "unresolved_issues_statuses":
+            widgetPropertiesToCreate.add(
+              new WidgetPropertyDto()
+                .setWidgetId(widget.getId())
+                .setPropertyKey(FILTER_PROPERTY)
+                .setTextValue(unresolvedIssues.getId().toString()));
+            widgetPropertiesToCreate.add(
+              new WidgetPropertyDto()
+                .setWidgetId(widget.getId())
+                .setPropertyKey(DISTRIBUTION_AXIS_PROPERTY)
+                .setTextValue("statuses"));
+            updateWidget(session, widgetIdsWithPropertiesToDelete, widget);
+            break;
+          default:
+            // Nothing to do, move along
+            break;
+        }
+      }
+
+      dbClient.widgetPropertyDao().deleteByWidgetIds(session, widgetIdsWithPropertiesToDelete);
+      dbClient.widgetPropertyDao().insert(session, widgetPropertiesToCreate);
+
+      dbClient.loadedTemplateDao().insert(new LoadedTemplateDto()
+        .setType(LoadedTemplateDto.ONE_SHOT_TASK_TYPE)
+        .setKey(getClass().getSimpleName()), session);
+
+      session.commit();
+    } finally {
+      MyBatis.closeQuietly(session);
+    }
+  }
+
+  private void updateWidget(DbSession session, List<Long> widgetIdsWithPropertiesToDelete, WidgetDto widget) {
+    dbClient.widgetDao().update(session,
+      widget.setWidgetKey(PROJECT_ISSUE_FILTER_WIDGET_KEY)
+        .setConfigured(true));
+    widgetIdsWithPropertiesToDelete.add(widget.getId());
```

Comment:
Hard to understand this "widgetIdsWithPropertiesToDelete" name. Why not "widgetIdsToDelete" ?
Moreover, add this id in this method is misleading.